src/rednoise.c

```diff
@@ -29,12 +37,13 @@ int main(int argc, char *argv[])
 #endif
 
     printf("\n\n");
-    printf("     Rednoise Removal Routine v2.0\n");
-    printf("            Feb, 2015\n\n");
+    printf("     Rednoise Removal Routine v2.0.1\n");
+    printf("                May, 2024\n\n");
```

Comment:
Nice.  I'm terrible at updating these and never remember to do it.  So they are all hopelessly out of date.